_maps_2024/12-jiatong-time&space.md

```diff
@@ -0,0 +1,15 @@
+---
+day: 12
+creator: Jiatong Su
+title: Territorial Changes of the Qing Dynasty
+thumbnail: /assets/thumbnails/2024/day12_time_space.png
```

Comment:
**request:** For consistency, would you do me a favor and change the name of the image file to `12-jiatong-time-space.png`?